identifiers/country-us/census_autogenerated/us_sldu.csv

```diff
@@ -1768,7 +1768,12 @@ ocd-division/country:us/state:vt/sldu:bennington,Vermont Bennington State Senate
 ocd-division/country:us/state:vt/sldu:caledonia,Vermont Caledonia State Senate district
 ocd-division/country:us/state:vt/sldu:grand_isle-chittenden,Vermont Grand-Isle-Chittenden State Senate district
 ocd-division/country:us/state:vt/sldu:chittenden,Vermont Chittenden State Senate district

```

Comment:
as with the sldl PR, please add validFrom and validTo dates fro districts that are being removed and added.